tests/insertion.rs

```diff
@@ -110,12 +110,10 @@ fn mapped_existing_entity() {
 
     server_app.connect_client(&mut client_app);
 
-    // Make client and server have different entity IDs.
```

Comment:
Client and server already have different IDs due to observers and now connected clients.
So I just added an assertion that they are different below.